mage_ai/frontend/oracle/icons/index.tsx

```diff
@@ -1644,3 +1644,20 @@ export const LayoutSplit = build([
   fill: 'none',
   viewBox: '0 0 16 16',
 });
+
+export const Warning = build([
```

Comment:
@anniexcheng We actually have this icon already. It's called `AlertTriangle` so you can remove the `Warning` icon since it's a duplicate.
![image](https://github.com/mage-ai/mage-ai/assets/78053898/7525ea21-d3b0-4ff4-86fe-ccccffc1fedd)